docs/_sass/_content.scss

```diff
@@ -0,0 +1,41 @@
+.docs {
+  main {
+    hr {
+      margin: 2rem 0 1rem 0;
+    }
+
+    h1 {
+      font-size: 2rem;
+      margin-top: 1.5rem;
+    }
+
+    h2 {
+      font-size: 1.5rem;
+      margin-top: 2.5rem;
+
+      &:first-child {
+        margin-top: 1.5rem;
+      }
+    }
+
+    h3 {
+      font-size: 1.25rem;
+      margin-top: 2.25rem;
+    }
+
+    h4 {
+      font-size: 1rem;
+    }
+
+    h5 {
+      font-size: 0.75rem;
+    }
+  }
+
+  pre {
+    background: #ececec;
+    border-radius: 5px;
```

Comment:
- You could use $gray-lighter here for the background-color
- Consider using `@include border-radius()` here instead of a fixed border-radius value
- rem's or em's would be preferred over px for padding